docker-compose.yml

```diff
@@ -30,6 +31,9 @@ services:
     # so define them in .override.yml and .windows.yml since target varies
     # either /var/lib/postgresql/data/ on Linux or c:\data on Windows
     # https://docs.docker.com/compose/extends/#adding-and-overriding-configuration
+    volumes:
```

Comment:
We can delete this comment now too.

This is awesome, love having only one compose file.

Reply:
Yeah, I was thinking I'd leave the warning about volume overrides for posterity, but maybe YAGNI on this one. I'm hopeful that we'll never deal with volume overrides again, but not positive yet.